configuration/printers/prusa-mk3s/display.cfg

```diff
@@ -9,6 +9,14 @@ d7_pin: display_d7_pin
 encoder_pins: ^display_encoder_pin1,^display_encoder_pin2
 click_pin: ^!display_click_pin
 
+[output_pin LCD_backlight_pin]
+pin: display_backlight_pin
+pwm: True
+hardware_pwm: True
+value: 0.01
+shutdown_value: 1
+cycle_time: 0.001
+
 [output_pin BEEPER_pin]
```

Comment:
Similar for this actually, this should also be removed (or moved to the printer template)